src/common/helpers/formatPriceRequestVoteData.ts

```diff
@@ -2,8 +2,9 @@ import { BigNumber, utils } from "ethers";
 import { PriceRequestRound } from "common/hooks/useVoteData";
 import formatRequestKey from "./formatRequestKey";
 import toWei from "common/utils/convertToWeiSafely";
+import { DateTime } from "luxon";
```

Comment:
Apparently, although I think a couple features don't work because they use some stuff from the window object.

https://moment.github.io/luxon/docs/manual/install.html#:~:text=If%20you%20want%20to%20work,setLocale()%20will%20do%20nothing.